sdk/PrebidServerAdapter/PBServerAdapter.m

```diff
@@ -133,6 +137,10 @@ - (NSDictionary *)requestBodyForAdUnits:(NSArray<PBAdUnit *> *)adUnits {
         adUnitConfig[@"sizes"] = sizeArray;
         
         adUnitConfig[@"config_id"] = adUnit.configId;
+        if (adUnit.adType == PBAdUnitTypeInterstitial && [[PrebidMobileDemandSDKLoaderSettings sharedInstance] isDemandEnabled:@"audienceNetwork"]) {
```

Comment:
I'm thinking about always pass instl =1 for interstitial type, that's by definition of open rtb. 
With handling interstitial size for facebook on the server side, you don't have to pass in size 0,0 here.